app/objects/player.py

```diff
@@ -549,12 +547,11 @@ async def silence(self, admin: "Player", duration: int, reason: str) -> None:
             {"silence_end": self.silence_end, "user_id": self.id},
         )
 
-        log_msg = f'{admin} silenced ({duration}s) for "{reason}".'
         await app.state.services.database.execute(
             "INSERT INTO logs "
-            "(`from`, `to`, `msg`, `time`) "
-            "VALUES (:from, :to, :msg, NOW())",
-            {"from": admin.id, "to": self.id, "msg": log_msg},
+            "(`from`, `to`, `action`, `msg`, `time`) "
+            "VALUES (:from, :to, 'silence', :msg, NOW())",
+            {"from": admin.id, "to": self.id, "msg": reason},
```

Comment:
''

Reply:
solved in https://github.com/cmyui/gulag/commit/6710da3c927f7afdf628e4f7d74d97314d27adc3